sdk/program/src/clock.rs

```diff
@@ -7,7 +7,7 @@
 //! by their slot number, and some slots do not contain a block.
 //!
 //! An approximation of the passage of real-world time can be calculated by
-//! multiplying a number of slots by [`SLOT_MS`], which is a constant target
+//! multiplying a number of slots by [`DEFAULT_MS_PER_SLOT`], which is a constant target
```

Comment:
Also, given that we are now linking to `DEFAULT_MS_PER_SLOT` in places like this, it would be nice if that constant had docs. Do you mind adding?

Reply:
Will do!